src/blocks/alert/Alert.tsx

```diff
@@ -0,0 +1,148 @@
+import type { ReactNode, FC } from 'react';
+import styled, { FlattenSimpleInterpolation } from 'styled-components';
+
+import type { TransformedHTMLAttributes } from '../Blocks.types';
+import type { AlertVariant } from './Alert.types';
+import { alertVariants } from './Alert.utils';
+import { Cross } from 'blocks/icons';
+import { HoverableSVG } from 'blocks/hoverableSVG';
+import type { ThemeColors } from 'blocks/theme/Theme.types';
+import { getTextVariantStyles } from 'blocks/Blocks.utils';
+
+export type AlertProps = {
+  /* Child react nodes rendered by Box */
+  children?: ReactNode;
+  /* Additional prop from styled components to apply custom css to Alert */
+  css?: FlattenSimpleInterpolation;
+  /* Sets the variant of the alert */
+  variant?: AlertVariant;
+  /* Boolean to set the alert as closable */
+  closeable?: boolean;
+  /* Close function to be called on close button click */
+  onClose?: () => void;
+  /* Boolean to set the alert as retryable */
+  retryable?: boolean;
+  /* Retry function to be called on retry button click */
+  onRetry?: () => void;
```

Comment:
this can be a prop like  action?: ReactNode; and  will a custom react node .
So instead of retryable, onRetry and retryText it will one custom action component

Reply:
According to the design, we'll not having custom components here, will only require a change of text message later on if required. Let's remove `retryable` and `closeable` and use the functions. Will reduce number of props being passed.

Reply:
@kalashshah it should have onClose?:()=>void and action:ReactNode prop only. We will be passing the Try again thing from outside. In figma Try again is only a reference and could change for certain use cases.

cc: @mishramonalisha76

Reply:
@kalashshah let's do actionText, onAction and onClose.

cc: @mishramonalisha76

Reply:
Done, please check